pkgs/kcp-dev/kcp/kubectl-kcp/registry.yaml

```diff
@@ -0,0 +1,28 @@
+packages:
+  - name: kcp-dev/kcp/kubectl-kcp
+    type: github_release
+    repo_owner: kcp-dev
+    repo_name: kcp
+    asset: kubectl-kcp-plugin_{{trimV .Version}}_{{.OS}}_{{.Arch}}.{{.Format}}
+    files:
+      - name: kubectl-kcp
+        src: bin/kubectl-kcp
+      - name: kubectl-ws
+        src: bin/kubectl-ws
+      - name: kubectl-workspaces
+        src: bin/kubectl-workspaces
+    format: tar.gz
+    description: kubectl plugins for kcp
+    overrides:
+      - goos: windows
+        files:
+          - name: kubectl-kcp.exe
+            src: bin/kubectl-kcp.exe
+          - name: kubectl-ws.exe
+            src: bin/kubectl-ws.exe
+          - name: kubectl-workspaces.exe
+            src: bin/kubectl-workspaces.exe
+    supported_envs:
+      - darwin
+      - linux
+      - windows
```

Comment:
```suggestion
```

If all platforms (darwin, linux, and windows) are supported, `supported_envs` isn't needed.